src/app.ts

```diff
@@ -14,7 +14,6 @@ import {
   amplitudePlugin,
   bugsnagPlugin,
   getRequestIdFastifyAppConfig,
-  healthcheckMetricsPlugin,
```

Comment:
most likely we won't need this as a separate plugin anymore, the logic will be incorporated into healthchecks themselves